### DISCUSSION TITLE
Matek f405 wing Accelerometer Barometer unavailable.

### DISCUSSION THREAD

**Discussion by 66nola66** [iNavFlight/inav#8651] [Q&A]
This may be a hardware issue but I have an F405, F722 and F765 that when flashed with 5.1 the accelerometer and Baro both show UNAVAILABLE when I pull a status on cli. I try to calibrate accelerometer but I press calibrate and it just looks to start then stops and never starts process. I tried to flash INAV 6 and same issue but where you calibrate the accelerometer it just has a red box that says "RESET ACCELEROMETER CALIBRATION" , If I his that it again just seems to start but reverts right back to the above message. Im kinda getting the feeling this is hardware unless I am just missing something here.

**PLEASE DELETE THE TEXT ABOVE AFTER READING AND UNDERSTANDING IT**

****

## Steps to Reproduce

1.
2.
3.
4.

## Additional context

---

- FC Board name and vendor:
- INAV version string:

**Comment by ABLomas** [comment]
F405-WING, F722 works for me without issues - including baro/acc right out of the box. Did you marked "reset to defaults" when flashing firmware?
